src/controller/DevicePairingDelegate.h

```diff
@@ -124,8 +124,6 @@ class DLL_EXPORT DevicePairingDelegate
      * The implementation should set the ICD registration info on the CommissioningParameters of the CommissioningDelegate
      * using CommissioningDelegate.SetCommissioningParameters(), and then call DeviceCommissioner.ICDRegistrationInfoReady()
      * in order to resume the commissioning process.
-     *
-     * The implementation may set the credentials before start commissioning, and call ICDRegistrationInfoReady() directly.
```

Comment:
This should probably document that if the ICD registration info is provided up front then this will not be called at all.

Reply:
I added. Thanks for your comment.